cmd/ctl/pkg/create/certificaterequest/certificaterequest.go

```diff
@@ -225,10 +255,44 @@ func (o *Options) Run(args []string) error {
 	}
 	req, err = o.CMClient.CertmanagerV1alpha2().CertificateRequests(ns).Create(context.TODO(), req, metav1.CreateOptions{})
 	if err != nil {
-		return fmt.Errorf("error creating CertificateRequest: %v", err)
+		return fmt.Errorf("error creating CertificateRequest: %w", err)
 	}
 	fmt.Fprintf(o.Out, "CertificateRequest %s has been created in namespace %s\n", req.Name, req.Namespace)
 
+	if o.FetchCert {
+		fmt.Fprintf(o.Out, "CertificateRequest %v in namespace %v has not been signed yet. Wait until it is signed...\n",
+			req.Name, req.Namespace)
+		err = wait.Poll(time.Second, o.Timeout, func() (done bool, err error) {
```

Comment:
This will check every second? That seems pretty fast for something that will likely take a few seconds to several minutes?

Reply:
Maybe you're right? I don't know what a good interval would be though. @munnerz @meyskens Thoughts?

Reply:
I don't think it's an issue for now - it may be a little over-eager, but I imagine most people will be using this command with non-ACME issuers (which are typically quite a bit quicker)